src/components/Callout/docs.md

```diff
@@ -0,0 +1,11 @@
+## Callout
+
+Callout styles for desktop and mobile.
+
+- `'expanded'`: boolean
+
+```react|responsive
+<Callout expanded setExpanded={() => undefined}>
```

Comment:
would be nice to habe a stateful example, maybe even with an discussion icon, where it can be clicked and and shows how to use and align it in the real world. The current example goes over the edge of the responsive container in the desktop version—hopefully that never happens in the real world.